packages/create-instance/create-slot-vnodes.js

```diff
@@ -18,7 +19,17 @@ function createVNodesForSlot (
   const el =
     typeof slotValue === 'string' ? compileToFunctions(slotValue) : slotValue
 
-  const vnode = h(el)
+  let vnode = h(el)
+  if (typeof slotValue === 'string') {
+    const vue = new Vue()
```

Comment:
I think this would be a problem if you mount a component that uses localVue and the slot component accesses a property that is only added to the localVue prototype, can you write a test to check?

Reply:
I will add test.

Reply:
Yes, it is.
Changing `localVue.prototype`, this does not work.
Using `vm._renderProxy` is correct if possible.
I will add test for changing `localVue.prototype`.